manifests/nodejs.yml

```
@@ -203,7 +203,7 @@ tests/:
       test_blocking.py:
         Test_Blocking: v3.19.0
         Test_Blocking_strip_response_headers: missing_feature
-        Test_CustomBlockingResponse: missing_feature
+        Test_CustomBlockingResponse: v5.15.0
```

This + needs an entry in the refs table at the start of the file :+1: 
```suggestion
        Test_CustomBlockingResponse: *ref_5_15_0
```